docs/devel/local-cluster/docker.md

```diff
@@ -225,6 +214,10 @@ On OS X, since docker is running inside a VM, run the following command instead:
 docker-machine ssh `docker-machine active` curl $ip
 ```
 
+## Deploy a DNS
+
+Read [documentation for manually deploying a DNS](http://kubernetes.io/docs/getting-started-guides/docker-multinode/#deploy-dns-manually-for-v12x) for instructions.
```

Comment:
Ok. I have no issues with this change. Its just that I have noticed very many links being broken even within the same repo. Thats why I expressed that concern.
This PR is fine as-is. May be we can consider maintaining a copy of the DNS doc here too?